src/components/cards/CreatePool/TokenPrices.vue

```diff
@@ -7,7 +7,7 @@ import useTokens from '@/composables/useTokens';
 import { computed } from 'vue';
 
 type Props = {
-  toggleUnknownPriceModal: () => void;
+  toggleUnknownPriceModal?: () => void;
```

Comment:
was getting a warning for this, even though its passed in? Making it optional quietens it up